lib/styles/text_style.dart

```diff
@@ -1,22 +1,22 @@
 import 'package:flutter/material.dart';
 
-TextStyle textStyleAppTitle() {
-  return const TextStyle(
-    fontSize: 32,
+TextStyle textStyleAppTitle(BuildContext context) {
+  return TextStyle(
+    fontSize: MediaQuery.of(context).size.width > 380 ? 32 : 22,
```

Comment:
what is 380 ?

Reply:
It's a Breakpoint under which there is an overflow